scripts/ci/pack_soljson.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+SOLJSON_JS="$1"
+SOLJSON_WASM="$2"
+SOLJSON_WASM_SIZE=$(wc -c "${SOLJSON_WASM}" | cut -d ' ' -f 1)
+OUTPUT="$3"
+
+(( $# == 3 )) || { >&2 echo "Usage: $0 soljson.js soljson.wasm packed_soljson.js"; exit 1; }
+
+# If this changes in an emscripten update, it's probably nothing to worry about,
+# but we should double-check when it happens.
+[ "$(head -c 5 "${SOLJSON_JS}")" == "null;" ] || { >&2 echo 'Expected soljson.js to start with "null;"'; exit 1; }
+
+TMPDIR=$(mktemp -d)
```

Comment:
Not that a temporary directory is nice - but stuff like the CI version of ``wat2wasm`` doesn't let me pipe through it at all.